src/t2iapi/device/service.proto

```diff
@@ -89,11 +91,11 @@ service DeviceService {
     rpc RemoveMdsDescriptor (BasicHandleRequest) returns (BasicResponse);
 
     /*
-    Trigger a descriptor update for the provided handle.
-    This manipulation of the device shall result in a msg:DescriptionModificationReport message with a report part with
-    modification type upt.
+    Trigger a descriptor update for the provided handles.
+    This manipulation of the device shall result in a msg:DescriptionModificationReport message with one report part with
+    modification type upt for each handle provided.
```

Comment:
They may appear in one report part that is Upt, not separate parts, which would still be sufficient.

```suggestion
    This manipulation of the device shall result in a msg:DescriptionModificationReport message in which each provided handle is present in a report part with modification type upt.
```

Reply:
Unfortunately, this is not sufficient. The Test case Biceps:R5025 checks the order of ReportParts where one ReportPart contains the parentHandle and the other one contains the childHandle. When both are contained in the same ReportPart (which is of course possible), there is no order that could be checked.

Reply:
If parent-child is not separated, it violates R5025 and a failure is fine. But if they are siblings, they are allowed to share a report part, since they do not have a parent-child relation. The current phrasing would require an SDC Stack to separate those as well, and I don't think that's implemented anywhere.